dev/archery/archery/crossbow/cli.py

```diff
@@ -285,6 +285,42 @@ def status(obj, job_name, fetch, task_filters):
     report.show(output)
 
 
+@crossbow.command()
+@click.option('--arrow-remote', '-r', default=None,
+              help='Set GitHub remote explicitly, which is going to be cloned '
+                   'on the CI services. Note, that no validation happens '
+                   'locally. Examples: https://github.com/apache/arrow or '
+                   'https://github.com/raulcd/arrow.')
+@click.option('--crossbow', '-c', default='ursacomputing/crossbow',
+              help='Crossbow repository on github to use')
+@click.option('--fetch/--no-fetch', default=True,
+              help='Fetch references (branches and tags) from the remote')
+@click.option('--github-token', envvar='ARROW_GITHUB_API_TOKEN',
+              help='OAuth token to create comments in the arrow repo. '
+                   'Only necessary if --track-on-pr-titled is set.')
+@click.option('--job-name', required=True)
+@click.option('--track-on-pr-titled', default=None,
+              help='Track the job submitted on PR with given title')
+@click.pass_obj
+def report_pr(obj, arrow_remote, crossbow, fetch, github_token, job_name,
+              track_on_pr_titled):
+    arrow = obj['arrow']
+    queue = obj['queue']
+    if fetch:
+        queue.fetch()
+    job = queue.get(job_name)
+
+    if track_on_pr_titled:
```

Comment:
Why is it optional?

Reply:
I was thinking that in the future we could add other ways of looking for the PR, like id but you are correct I have made it mandatory as right now it should always be present.